### PR TITLE
add(epp): Polycom DMA/HDX Monit Proc EN

### PR DESCRIPTION
**Pull request by thibaults-centreon**
New monitoring procedure for Polycom devices.
FR will follow later

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+| Metric name                               | Description                                    | Unit   |
+|:----------------------------------------- |:-----------------------------------------------|:-------|
+| dma.conferences.active.count              | Number of active conference on the DMA         | Count  |
+| dma.cluster.conferences.active.count      | Number of active conference on a cluster       | Count  |
+| dma.cluster.participants.active.count     | Number of active participants on a cluster     | Count  |
+| dma.cluster.local.database.users.count    | Number of local users in a cluster database    | Count  |
+| dma.cluster.custom.conference.rooms.count | Number of custom user rooms in a cluster       | Count  |
+| dma.cluster.video.port.usage.count        | Number of video ports used by a cluster        | Count  |
+| dma.cluster.video.port.free.count         | Number of free video ports on a cluster        | Count  |
+| dma.cluster.video.port.percentage         | Percentage of video port used by a cluster     | Count  |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md:48]:
```suggestion
| dma.cluster.video.port.percentage         | Percentage of video port used by a cluster     | %  |
```

File: en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+| dma.cluster.voice.port.usage.count        | Number of voice ports used by a cluster        | Count  |
+| dma.cluster.voice.port.free.count         | Number of free voice ports on a cluster        | Count  |
+| dma.cluster.voice.port.percentage         | Percentage of voice port used by a cluster     | Count  |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md:51]:
```suggestion
| dma.cluster.voice.port.percentage         | Percentage of voice port used by a cluster     | %  |

```

File: en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Alerts-->
+
+| Metric name             | Description        | Unit  |
+| :---------------------- | :----------------- | :---- |
+| dma.alerts.total.count  | Number of alerts.  | Count |
+
+Specify through the `--warning-status` and `--critical-status` options which severities
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md:34]: That doesn't "brake" the "DOCUSAURUS_CODE_TABS" ?

File: en/integrations/plugin-packs/procedures/hardware-devices-polycom-hdx-snmp.md [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+| memory.buffer.bytes     | Buffered Memory allocation.               |   B   |
+| memory.cached.bytes     | Cached Memory allocation.                 |   B   |
+| memory.shared.bytes     | Shared Memory allocation.                 |   B   |
+
+<!--Traffic-->
+
+| Metric name                         | Description                                   | Unit    |
+| :---------------------------------- | :-------------------------------------------- | :------ |
+| status                              | Status of the interface                       |         |
+| interface.traffic.in.bitspersecond  | Incoming traffic going through the interface. | B/s & % |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/hardware-devices-polycom-hdx-snmp.md:56]: Only B/s ?
(Event if the threshold can be set on %.

File: en/integrations/plugin-packs/procedures/hardware-devices-polycom-hdx-snmp.md [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+| interface.traffic.out.bitspersecond | Outgoing traffic going through the interface. | B/s & % |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/hardware-devices-polycom-hdx-snmp.md:57]: Same remark.

File: en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+
+## Host configuration 
+
+* Add a new Host and apply the *HW-Device-Polycom-Dma-SNMP-Custom* Host Template
+* Fill SNMP Version and Community fields according to the device's configuration
+
+
+  :warning: When using SNMP v3, use the SNMPEXTRAOPTIONS Macro to add specific authentication parameters
+
+| Obligatoire | Nom              | Description                                    |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md:153]: Français here.

File: en/integrations/plugin-packs/procedures/hardware-devices-polycom-hdx-snmp.md [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Host configuration 
+
+* Add a new Host and apply the *HW-Device-Polycom-Hdx-SNMP-Custom* Host Template
+* Fill SNMP Version and Community fields according to the device's configuration
+
+  :warning: When using SNMP v3, use the SNMPEXTRAOPTIONS Macro to add specific auth parameters
+
+| Obligatoire | Nom              | Description                                    |
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/hardware-devices-polycom-hdx-snmp.md:144]: Français here.

File: en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+| dma.server.swap.usage.percentage      | Swap usage of a server                    |   B   | 
+| dma.server.swap.free.bytes            | Free swap on a server                     |   B   |
+| dma.server.swap.usage.percentage      | Swap percentage use on a server           |   %   |
+| dma.server.disk.usage.bytes           | Disk space usage of a server              |   B   |
+| dma.server.disk.free.bytes            | Free disk space on a server               |   B   |
+| dma.server.disk.usage.percentage      | Disk percentage space usage on a server   |   %   |
+| dma.server.logs.usage.bytes           | Logs space usage of a server              |   B   |
+| dma.server.logs.free.bytes            | Free logs space on a server               |   B   |
+| dma.server.logs.usage.percentage      | Logs percentage space usage on a server   |   %   |
+
```
Comment (pkriko) [inline en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md:93]:
```suggestion

<!--END_DOCUSAURUS_CODE_TABS-->

```

File: en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md (hunk 3 above)
Comment (thibaults-centreon) [reply en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md:34]: No, this should be ok as for https://raw.githubusercontent.com/centreon/centreon-documentation/master/en/integrations/plugin-packs/procedures/operatingsystems-linux-snmp.md for example

**Pull request merged**